addons/website/views/website_views.xml

```diff
@@ -241,9 +241,9 @@
                     <field name="url"/>
                     <field name="website_id" groups="website.group_multi_website"/>
                     <group string="Group By">
-                        <filter string="name" name="group_by_name" domain="[]" context="{'group_by':'name'}"/>
-                        <filter string="url" name="group_by_url" domain="[]" context="{'group_by':'url'}"/>
-                        <filter string="website"  name="my_websites" domain="[]" context="{'group_by':'website_id'}"/>
+                        <filter string="View Name" name="group_by_name" domain="[]" context="{'group_by':'name'}"/>
```

Comment:
thx...

But why 'View Name' ?

Reply:
My Bad, Updated.

Reply:
Sorry but I will prefer name ;)
You are in the tree view of menu, so what you do is group by Name really ;)

Reply:
Indeed, We should remove name from Group by. 
I kept it as It was already there in previous versions.


What do you think?

Reply:
@JKE-be  Any feedback/Improvement? (or Better if this can be handled in any BacktoBasic task)